### PR TITLE
GJ-452: adding external rec. api docs 'instrument reporting'

### PR DESCRIPTION
**Pull request by victokoh**
See title, can someone proofread this please?

### CODE REVIEW

File: source/includes/_instruments.md [hunk 1]
```diff
@@ -327,3 +327,165 @@ Http Status:
 | 400 Bad Request  | The request was malformed.  See response body    | None       | n/a              |
 | 401 Unauthorized | No auth provided, auth failed, or not authorized | None       | n/a              |
 | 404 Not Found    | No allocation with this UUID exists              | None       | n/a              |
+
+
+
+## Instrument Reporting
```
Comment (edslocombe) [inline source/includes/_instruments.md]: My view would be to rename to 'Asset custody reconciliation'.

Comment (edslocombe) [reply source/includes/_instruments.md]: And perhaps make it a section (e.g. `<em>Asset custody reconciliation</em>`)

File: source/includes/_instruments.md [hunk 2]
```diff
@@ -327,3 +327,165 @@ Http Status:
...
+
+The [`POST /instruments/reporting`](/#settlement-equity-post-instruments-reporting) endpoint allows platforms to report their system's view on equity ownership for reconcillation purposes.
```
Comment (edslocombe) [inline source/includes/_instruments.md]: Be careful using the word ownership here. Not something I pretend to know lots about but there could be confusion in that for at least PP, this is _beneficial_ ownership.

Perhaps _allocation_ is simpler.

File: source/includes/_instruments.md [hunk 3]
```diff
@@ -327,3 +327,165 @@ Http Status:
...
+⚠️ All instruments that are part of the same snapshot should share the same `reportId`.
+
+
+## Instrument Reporting Model
+
+### Required Attributes
+
+All of the following attributes of the Instrument Reporting Model are considered 'required' attributes. 
+
+⚠️ In the case of an instrument that has not yet been issued or traded to investors - the holdings section can be left empty.
```
Comment (edslocombe) [inline source/includes/_instruments.md]: Will it even need to be reported at all? Either way fine but could be useful clarifying.

File: source/includes/_instruments.md [hunk 4]
```diff
@@ -327,3 +327,165 @@ Http Status:
...
+
+
+| Key                  | JSON Type | Value Type       | Value Description                                                                                                      |
+|----------------------|-----------|------------------|------------------------------------------------------------------------------------------------------------------------|
+| reportId             | String    | UUID             | A platform-generated UUID for this equity ownership snapshot, shared between all instruments included in this snapshot.|
+| instrumentSymbol     | String    | InstrumentSymbol | The unique ID/symbol of the instrument registered by the platform.                                                     |
+| timestamp            | String    | ISO 8601         | The time at which the snapshot of all instruments' holdings was originally taken.                                      |
```
Comment (edslocombe) [inline source/includes/_instruments.md]: Did we agree in the first instance that this must be `<date>T23:59:59.999Z`?

File: source/includes/_instruments.md (hunk 1 above)
Comment (edslocombe) [reply source/includes/_instruments.md]: We could flesh this out more with instructions for the platform to submit on every working day before 9am for example (whilst still accepting on weekends / holidays).

File: source/includes/_instruments.md [hunk 5]
```diff
@@ -327,3 +327,165 @@ Http Status:
...
+POST /instruments/reporting HTTP/1.1
+Host: api-sandbox.goji.investments
+Content-Type: application/json
+Authorization: Basic ...
+X-GOJI-CLIENT-ID: 59425d3d-cf73-44ff-aecb-590cd198a4bc
+
+{
+  "reportId": "68f54225-6760-449d-8f38-a003d1dc535a",
+  "instrumentSymbol": "SPV1",
+  "timestamp": "2020-01-09T08:53:20.963Z",
```
Comment (edslocombe) [inline source/includes/_instruments.md]: This worth updating assuming my previous comment is correct?

File: source/includes/_instruments.md (hunk 2 above)
Comment (victokoh) [reply source/includes/_instruments.md]: sure, makes sense

File: source/includes/_instruments.md (hunk 3 above)
Comment (victokoh) [reply source/includes/_instruments.md]: i think it's probably clearer if they just report all instruments regardless.

File: source/includes/_instruments.md (hunk 1 above)
Comment (victokoh) [reply source/includes/_instruments.md]: oki

File: source/includes/_instruments.md (hunk 4 above)
Comment (victokoh) [reply source/includes/_instruments.md]: was wondering if it was platform-specific, so i didn't specify that here.

File: source/includes/_instruments.md (hunk 5 above)
Comment (victokoh) [reply source/includes/_instruments.md]: sure

File: source/includes/_instruments.md (hunk 4 above)
Comment (victokoh) [reply source/includes/_instruments.md]: added

**Pull request merged**